pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -173,6 +173,12 @@
     public static final String SEGMENT_RELOCATOR_INITIAL_DELAY_IN_SECONDS =
         "controller.segmentRelocator.initialDelayInSeconds";
 
+    // configs for uploading missing LLC segments to deep store
+    public static final String ENABLE_DEEP_STORE_RETRY_UPLOAD_LLC_SEGMENT =
```

Comment:
Can you please add a paragraph (liberal comments with examples. they are zero cost)  before each config you have introduced? It will help a lot when we stare at this code six months down the road, or when we start to look at some behavior in production.